src/pages/trade/ui/chart.tsx

```diff
@@ -173,6 +173,17 @@ const ChartData = observer(({ candles }: { candles: OhlcData[] }) => {
     }
   }, [candles]);
 
+  // Handle window resize to re-fit content
+  useEffect(() => {
+    const handleResize = () => {
+      if (chartRef.current) {
+        chartRef.current.timeScale().fitContent();
+      }
+    };
+    window.addEventListener('resize', handleResize);
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
```

Comment:
TradingView is kinda buggy with window resizing. This guarantees the content fits when it hits those breakpoints.